packages/flutter/lib/src/foundation/persistent_hash_map.dart

```diff
@@ -0,0 +1,388 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A collection of key/value pairs, from which you can retrieve a value
+/// using its associated key.
+///
+/// This class implements a persistent map: extending this map with a new
+/// key/value pair does not modify an existing instance but instead creates a
+/// new instance.
+///
+/// Note: unlike [Map] this class does not support `null` as a key value and
```

Comment:
remove "Note", see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
I'd argue that  "In particular, avoid saying "Note:". It adds nothing." is misguided. It says that people should pay a special attention to something. 

Anyway. Removed.